test-data/encrypted-envelops.test.json

```diff
@@ -18,5 +18,15 @@
             "version": "",
             "encryptionScheme": "x25519-chacha20-poly1305"
         }
+    },
+    "deliveryInformation": {
+        "ciphertext": "mSTXIS7LCebwZgxz+XtqtB3bobDQpfnh6jah3ciJ+H87c0D4HIQtumCmRssGhSAoMOG0MkYIYEeFw976MgvJNmyxMB4B60hL3z2F9fm+uNyhENlPhO+6ol8VVXSS8SiQObvqhnSk4p8Gqi5nF1lYZGz5c9+TSm9rccePxzyMzHHmI8r7+qs94JGB9FhKUxxW3rsMcfIAoq+SoAXp8flGTpwqSgK72u/7skb/LhyDwdMoIjYk5cu0I4+xEAw1VqllhsllbGmB5zAVHLrhDKaBH2lclMXgbN7ovQLLepfYpRZFE94AuuZowa/CQ3kHwbqTBtR8xYjDKkU7VpcpQTsWcLCBRdT8rBx9PL469JJ/viF4xOuDcHEMT9GrDVwbWOVgN52X/+7X2dk5Z0lQw9Q+ZsruL501Yo/fFMfFzNjtfe3GyMKEtU0mwMBYN4VTXZAh6ziPNf2lqv0O4QYTW2XcNhYNw1BLuCXiI5OHF0CPMHgaNP70RuCmGBs3Orml2wfNrGkxGVDMLWubbSmqNmmZjWflqLyXdlc79yZgkEtPqc0T/UtSw2zIj4RQVdW0N98k2PobW72A3JfT/oPrEzJuvF3N8M/Lr/JtoQhu+1PgLnhWr/lDUA6SdS4PgXAsQGmqY+s33JUVighUtLxP37NjfcQeoMU1/NojeAkxnckvK71sHk+hEHzOjTDtCip4mIf/k/TVhRdD+4Rd2xHFWxTyGsezxmueg3y3mmIq/gs4blNfnBIuFTFaEkKEfHs7WCoGDgyzKsyKXNKm2Jwz/UJiGLKIrlaPUNXfzLjJtmjIloLxDc/rFy+gfe9YbeTOZwRLei7z7HqL5W+bFQsWehAjLnviLuX7xLaZ48q5xThl0CgN0/r6nIW9iM12ukct9LhK0qH+EMifjaNGuBhMfRtn+aEkiLkkvb84KOzgtaaKCuBLT8dlvxhPyBbSTQ8re7poKEMBUYIPLDp/BhmnLpiqyEf0cAtqFNb9LWiB8sfuIlLqi26+PMODAsnuFMUiOJVnClMe5Hh1ZRKa2nGYzKWVYF5NX/RNwzAXR9jxrBNxk+WAAmb8OZD0DHYYhbgo9wQH5Wvfsodg2OTacAprcZoxi0BOUmYXh5Q0qId0gtl+aMT9xd9GRScg0HNVo9BoHlMnL1KoVu9exxF0WmXPrsE8e2hg/JClEn1TpWetrtXWDQvyltu+6GhHu3oHZW3mTx5SkGKTG7whJv9g3B/ZbPC/mI/fgvxSVOQDYL9V+FPtgF9cLCsna1AEiD6dEgeKCtiw6BN6kYiqDAl0lbjuvV/eS44jwrGmj3a2Mi5K+OxWEJMXPgMAlrTRAutyfaiaSZXrjeFF8/S1yciaqBrxnk3pKVIeWDuQF0lcP7vq55aW7XWOQbTL2V455kJfgXwGkrhJjK3U8TFG91cXiZWA3OZfVnSXHD4JPd1EEp6j7aKziPScxnq7XcXiC6QZyqHo3hPDVigVE2b6frCG87Z/xfqgvGYR4x3dmaEMoZDMmWVsX8T0jcE4xWwMAyLpXbLv3w+TFP80EPFUyJRHEGe0voHCOCM9eSIJKkSpGXlsZSr4BISMegfjdl2/ORC0Nr2kYexnYFKu5TQ8jD/zmP9T2uf6aZMGtia3V6lHRDCZ/g99WuBUS072VP0rymErh/NhLJp+XsPJSowy+Rqj5lsDHviF33BoVKJggHuIc29ZuBoOpJ7rdGWIf8g6qWIdTZ3e4MBE4qUMM33WjrR22sayKYCuZyrMxYZZu0FKZG2iApi4G2jNiOSty7oBK1mZs0tB1ynGQcUkakFoGgZ5aM5c5EHJM53btIcFBVSvvehtobVBfOOPKjNVDJSdSZixAhW83kmZZZAyOzpoqSN2picEsQvjincxGKw7iBLrUUqIQXQMmi4QXGHM+Bn2eUxgqL4F57+QHLDJhxXgTCwHy8MNgXDra4gOxu2C+c6cMxjrXHitaj2IRnFkhV0aK9MoDL4Ze0zSGiLRzn4q/Ck5q9ecGVo+hGCDse6rt9HZ1RdxfKGKhAYd30Cf7vpZpVqWRzqRgV+FD8luN9W8QgwiwXic6+M1cvuot6da7aygrxtJOgFK2lHuvQpRHuE5iF7mhNnAx5Imap8rQ0j4GMpRQKwDQebhaa8eUfhMOafHVYRfKgtIvaxvmYMhMxZXxwKxxzGR6UmjdqPClbRjn6nT/+RjHN05FkZQBNmJBFqjVMWoydAGdWJLz5GyfMKwGERtsDu7jV7g38qH3mfjAl5cO21K/qcs1xKj0vawQ8TuUl1PszsRO58ouUsSKmrkyKCgJYvbA7tJksEpqIC7YD73ZAWY4/D16cnYsiEfKcW9dtcaeqTN4pK8CJV4oiZSZDDL9MyVM5kYODxmVk2uM3sH40gtp1cLn0F3I3mAdbVQR+ulVf2LgCbWRCQrvdio/Yki7CniSRqIqOBGRvQQgFRbQ0edZclwlH+55FMlpEB1d6oE17kTjv//gUNrqUoL3jTYPwT7N6Rb5xYVBjdXO1FgW1E6mixDpSiaG+4CJ0FQpswn874PAA8vhaifzNyAmS9zbj24nDkQdaBX4ntET2TwVXlcpIn1RmxGuXV7NvdZa12tF1eGEda9iSeF+88b48z0ykd8vmAInJCqHS1Bw1shmKxXZtwyNrl61VM6GT46LKBkXez9cVY/wo9BJ+IEQwtrqQFFfjho+xYm8rsDqf7xe6k+5yXHe6WrCuQ+5SEB259HmGNJPI6U4apU4kMfa/EZ9GaAfBHG",
```

Comment:
Maybe it could help us in the future to add a comment which explains which structure the decrypted object has and what keys were being used in order to encrypt it.